legal-api/src/legal_api/models/business_common.py

```diff
@@ -131,7 +131,7 @@ def entity_type(self):
         if self.is_legal_entity:
             return self._entity_type
 
-        if self.name_type.value != AlternateName.NameType.OPERATING:
+        if self.is_alternate_name_entity and self.name_type.value != AlternateName.NameType.OPERATING:
             return None
```

Comment:
This was throwing `AttributeError: 'LegalEntityHistory' object has no attribute 'name_type'`. I added an explicit check to make sure it is an alternate name entity.